src/gg.regression.unity.bots/Tests/Scripts/GameScene/GameSceneTests.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections;
+
+using NUnit.Framework;
+using RegressionGames.StateRecorder;
+using RegressionGames.StateRecorder.Types;
+using RegressionGames.TestFramework;
+using RegressionGames.Types;
+#if UNITY_EDITOR
+using UnityEditor;
+using System.Reflection;
+#endif
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+using Object = UnityEngine.Object;
+// ReSharper disable CheckNamespace
+// ReSharper disable InconsistentNaming
+namespace RegressionGames.Tests.Z_RunMeLast_GameScene
+{
+    [TestFixture]
+
+    public class GameSceneTests
+    {
+        private int oldIndex = -1;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            SavePreviousViewAspect();
+            //force this to match our recording resolution
+            Screen.SetResolution(1920, 1080, false);
+
+            // set to full HD (1920x1080)
+            SetEditorAspectRatio(3);
+
+            MouseEventSender.InitializeVirtualMouse();
+        }
+
+        [UnitySetUp]
+        public IEnumerator SetUp()
+        {
+            var botManager = Object.FindObjectOfType<RGBotManager>();
+            if (botManager != null)
+            {
+                // destroy any existing overlay before loading new test scene
+                Object.Destroy(botManager.gameObject);
+            }
```

Comment:
I saw this in a few places, just out of curiosity why does this need to be removed? I would think that since we are loading a new scene, this would be removed anyways but I'm guessing you found this was not the case?

Reply:
dontdestroyonload - persists across scene loads, even in testing